confluent_client/producer.py

```diff
@@ -0,0 +1,47 @@
+from confluent_kafka import Producer
+from utils import config
+
+class KafkaProducer:
+    '''
+    Kafka producer class to produce messages to a Kafka topic using Confluent Kafka Python client.
+        
+    Attributes:
+        producer (Producer): Confluent Kafka producer object.
+        
+    Methods:
+        delivery_callback: Callback function to handle message delivery status.
+        produce: Produce a message to a Kafka topic.
+        
+    '''
+    
+    config = {
+        'bootstrap.servers': config.KAFKA_BOOTSTRAP_SERVER,
+        'sasl.username': config.KAFKA_API_KEY,
+        'sasl.password': config.KAFKA_API_SECRET,
+        'security.protocol': config.KAFKA_SECURITY_PROTOCOL,
+        'sasl.mechanisms': config.KAFKA_SASL_MECHANISM,
+        'acks': config.KAFKA_ACKS
+    }
+    
+    def __init__(self, config):
+        self.producer = Producer(config)
```

Comment:
@kevklash , Let's make this Feature Flag enabled.  That way it doesn't try to run this code when someone does local development.   For an example of this, take a look at the gridfs code, where it uses grid if it see's the LD flag, else it just loads the local json.